runtime/gc_base/GCExtensions.hpp

```diff
@@ -309,6 +309,8 @@ class MM_GCExtensions : public MM_GCExtensionsBase {
 	 */
 	static bool needScanStacksForContinuationObject(J9VMThread *vmThread, j9object_t objectPtr, bool isGlobalGC);
 
+	bool isConcurrentScavengerInActive(MM_EnvironmentBase* env);
```

Comment:
put a comment above that it's valid to use this from both mutator threads and GC threads